lib/android/nit_activity.nit

```diff
@@ -279,6 +286,11 @@ class Activity
 	# Notification from Android, the current device configuration has changed
 	fun on_configuration_changed do end
 
+	# The back key has been pressed
+	#
+	# Return `true` if the event has been handled.
+	fun on_back_pressed: Bool do return false
```

Comment:
why does this return false by default ? if it's meant to be refined, why not make this function abstract ?